model/config.go

```diff
@@ -876,6 +876,7 @@ type ExperimentalSettings struct {
 	CloudUserLimit                  *int64  `access:"experimental,write_restrictable,cloud_restrictable"`
 	CloudBilling                    *bool   `access:"experimental,write_restrictable,cloud_restrictable"`
 	EnableSharedChannels            *bool   `access:"experimental"`
+	ThreadAutoFollow                *bool   `access:"experimental"`
```

Comment:
@reflog discussing with Jesus, can we please add this to the service settings section of the config?

Reply:
@esethna  done